connectors-home/modules/ROOT/pages/introduction/intro-config-use-acb.adoc

```diff
@@ -0,0 +1,220 @@
+= Using Anypoint Code Builder to Configure a Connector
+
+Create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and configure attributes. 
+
+== Before You Begin
+
+Before creating an integration project, you must:
+
+* xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
+* Have credentials to access the connector's API
+
+To use connectors, you must be familiar with:
+
+* The connector’s API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+
+After you complete the prerequisites, you are ready to create an app and configure the connector using Anypoint Code Builder.
+
+== Creating a Configuration File
+
+A configuration file is a YAML file that references other configurations and specifies values for your connector's properties. Define the configuration properties based on the environment you are deploying your application to.
+
+. In the *Explorer* view, right-click the *resources* folder (`src/main/resources`) and select *New File*.
+. Name the file `config.yaml`.
+. Populate your file with the connection properties for your connector, for example, your Salesforce, Gmail, and Slack configurations:
```

Comment:
"configurations" doesn't fit here.

Maybe: Populate your file with the properties required to connect to your connector?